### ISSUE TITLE
Highlight line is barely visible when switching from Console

### ISSUE THREAD

**Issue by digitarald** [devtools-html/debugger.html#4318]
Clicking on an error from Console, the highlight on the target line is barely visible as it fades out in the blink of an eye (300-400ms).

Ideas beyond just delaying and slowing the animation:

- Only start the fadeout only after the debugger fully loaded
- Only start the fade when the mouse moved significantly.

**Comment by jasonLaster** [comment]
closing as it's a dupe of #4100

**Comment by digitarald** [comment]
This issue was about the animation, not the missing highlight. Opened a new issue in [bugzilla](https://bugzilla.mozilla.org/show_bug.cgi?id=1544930) to continue the discussion.
